lpcompiler.pas

```diff
@@ -1638,9 +1647,9 @@ function TLapeCompiler.ParseMethod(FuncForwards: TLapeFuncForwards; FuncHeader:
     end;
 
     try
-      if (Tokenizer.Tok = tk_kw_Overload) or (FuncHeader.isOperator and (Tokenizer.Tok <> tk_kw_Override)) then
+      if (Tokenizer.Tok = tk_kw_Overload) or ((FuncHeader.MethodDef in [mdOperator, mdProperty]) and (Tokenizer.Tok <> tk_kw_Override)) then
       begin
-        if not FuncHeader.isOperator then
+        if not(FuncHeader.MethodDef in [mdOperator, mdProperty]) then
```

Comment:
What if you explicitly add `overload` after an operator/property? It must be without semicolon..?